src/inner-slider.js

```diff
@@ -704,7 +714,6 @@ export class InnerSlider extends React.Component {
       onMouseUp: touchMove ? this.swipeEnd : null,
       onMouseLeave: this.state.dragging && touchMove ? this.swipeEnd : null,
       onTouchStart: touchMove ? this.swipeStart : null,
-      onTouchMove: this.state.dragging && touchMove ? this.swipeMove : null,
```

Comment:
do you also need to preserve this logic around `this.state.dragging`?

Reply:
I've added it back in but in a slightly different manner to work around another iOS Safari quirk (touch listeners registered by other touch listeners don't seem to be able to be made non-passive).